src/BuildScriptGeneratorCli/Commands/CommandBase.cs

```diff
@@ -43,6 +51,14 @@ public int OnExecute(CommandLineApplication app, IConsole console)
                 if (envSettings != null && envSettings.GitHubActions)
                 {
                     logger?.LogInformation("The current Oryx command is being run from within a GitHub Action.");
+                    var githubActionBuildStartTime = GitHubBuildStartTime;
+                    var githubActionBuildEndTime = GitHubBuildEndTime;
+                    var buildEventProps = new Dictionary<string, string>()
+                    {
+                        { "githubActionBuildStartTime", githubActionBuildStartTime},
+                        { "githubActionBuildEndTime", githubActionBuildEndTime},
+                    };
+                    logger.LogEvent("GithubActionTimeLog", buildEventProps);
```

Comment:
We should have appropriate name for this event too

Reply:
Gotcha, I'll think of a better name:)